src/DynamoCoreWpf/Interfaces/IBrandingResourceProvider.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System.Windows.Media;
+
+namespace Dynamo.Wpf.Interfaces
+{
+    public enum ResourceName
+    {
+        StartPageLogo,
+        AboutBoxLogo
+    }
+
+    public interface IBrandingResourceProvider
+    {
+        ImageSource GetImageSource(ResourceName resourceName);
+        string GetStringResource(ResourceName resourceName);
```

Comment:
To align with the other API method `GetImageSource`, this method should have the form of `GetXxxYyy` where `XxxYyy` is the type this method returns (without the word `Resource` since that is implied by the interface name "Resource Provider"). Please rename this method and I'll merge it right in!

(Sorry for being pain in the butt but this is a public contract, I'll be more nitpicking than my usual self)